src/widget/wtracktableview.cpp

```diff
@@ -844,6 +844,44 @@ void WTrackTableView::keyPressEvent(QKeyEvent* event) {
     }
 }
 
+void WTrackTableView::resizeEvent(QResizeEvent* event) {
+    // When the tracks view shrinks in height, e.g. when other skin regions expand,
+    // and if the row was visible before resizing, scroll to it afterwards.
+
+    // these heights are the actual inner region without header, scrollbars and padding
+    int oldHeight = event->oldSize().height();
+    int newHeight = event->size().height();
+    bool maybeScrollTo = newHeight < oldHeight;
+
+    QModelIndex currIndex;
+    int rHeight = rowHeight(0);
+    if (maybeScrollTo) {
```

Comment:
```suggestion
    if (!maybeScrollTo) {return;}
```
The else branch is not needed, if it returns early.